### PR TITLE
update mycockpit myapps myprofil

### PR DESCRIPTION
**Pull request by Erriep**

### CODE REVIEW

**Comment by Erriep** [comment]
fixed bug on the page when user don't exist and screen a box to link on login connect

**Review by AntoineCAU — DISMISSED** [review]

File: src/components/Cockpit/index.jsx [hunk 1]
```diff
@@ -8,72 +8,88 @@ function Cockpit() {
...
+          if (!data.country || !data.zipcode || !data.city || !data.mobile)
+            setCompleted(false);
+        });
+      axios
+        .get(`${process.env.REACT_APP_API_URL}/appfav/${user.id}`)
+        .then(({ data }) => {
+          setMyApps(data);
+        });
+    }, []);
+  }
```
Comment (LoicBrassart) [inline src/components/Cockpit/index.jsx]: You should reverse this: _When my component is mounted, if `user` is not `null`, then blablabla_

File: src/components/MyApps/index.jsx [hunk 2]
```diff
@@ -7,42 +7,56 @@ import User from '../../contexts/User';
...
-  }, []);
+  if (user !== null) {
+    useEffect(() => {
+      axios
+        .get(`${process.env.REACT_APP_API_URL}/appfav/${user.id}`)
+        .then(({ data }) => {
+          setMyApps(data);
+        });
+    }, []);
+  }
```
Comment (LoicBrassart) [inline src/components/MyApps/index.jsx]: Same

File: src/components/MyProfile/index.jsx [hunk 3]
```diff
@@ -8,112 +8,126 @@ function MyProfile() {
...
+
+  if (user !== null) {
+    useEffect(() => {
+      axios
+        .get(`${process.env.REACT_APP_API_URL}/users/${user.id}`)
+        .then(({ data }) => {
+          setInfos(data);
+        });
+    }, []);
+  }
```
Comment (LoicBrassart) [inline src/components/MyProfile/index.jsx]: Same

File: src/components/MyProfile/index.jsx [hunk 4]
```diff
@@ -8,112 +8,126 @@ function MyProfile() {
...
+                  </div>
+                  <div className="pic">
+                    <img
+                      src="https://us.123rf.com/450wm/dstarky/dstarky1702/dstarky170200652/71441287-icône-de-vache-ou-d-un-logo-dans-le-style-de-ligne-moderne-haute-qualité-contour-noir-pictogramme-pour.jpg?ver=6"
+                      alt=""
+                      style={{
+                        width: '70px',
+                        height: '70px',
+                        borderRadius: '100%',
+                      }}
```
Comment (LoicBrassart) [inline src/components/MyProfile/index.jsx]: May I know why these styles are not elsewhere ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Evils to fix !

**Review by AntoineCAU — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**